app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncResponse.java

```diff
@@ -0,0 +1,80 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_response")
+public class SenderSyncResponse extends AbstractEntity {
+	
+	public static final long serialVersionUID = 1;
+	
+	public enum SyncResponseStatus {
```

Comment:
I don't think this class should have a status, after a response is processed, delete it instead

Reply:
Implemented.